src/__deprecated__/components/form/__definition__.js

```diff
@@ -1,6 +1,6 @@
 import Form from './';
-import Definition from './../../../demo/utils/definition';
-import OptionsHelper from 'utils/helpers/options-helper';
+import Definition from '../../../../demo/utils/definition';
+import OptionsHelper from '../../../utils/helpers/options-helper'
```

Comment:
Semicolon `;` at the end of this line has been lost.

ESLint's [`semi` rule](https://eslint.org/docs/rules/semi) should've detected this:

:small_blue_diamond: https://github.com/Sage/carbon-factory/blob/v8.0.2/.eslintrc#L80
```js
"semi": ["error", "always"]
```

This is another instance of the FE-1870 issue :cry: